CovertInfiltration/Src/CovertInfiltration/Classes/UISS_CovertActionRisks.uc

```diff
@@ -21,6 +21,11 @@ simulated function InitRisks()
 	Text.SetWidth(-X);
 }
 
+simulated function MoveUnderResourceBar(bool NewValue)
+{
+	SetY(NewValue ? 100 : 0);
+}
```

Comment:
`SetUnderResourceBar`? The name doesn't make it sound like it can, in fact, be used to move it up again.

Reply:
Ultra temporary solution. Will get fixed when the risks/deterrence/etc gets (re)done